src/components/card/card-dialog/CardDialog.js

```diff
@@ -0,0 +1,84 @@
+import { useState } from 'react';
+import Button from '@mui/material/Button';
+import Dialog from '@mui/material/Dialog';
+import DialogActions from '@mui/material/DialogActions';
+import DialogContent from '@mui/material/DialogContent';
+import DialogContentText from '@mui/material/DialogContentText';
+import DialogTitle from '@mui/material/DialogTitle';
+import { TextField } from '@mui/material';
+import PropTypes from 'prop-types';
+
+function CardDialog({ params }) {
+  const [open, setOpen] = useState(false);
+  const [title, setTitle] = useState(null);
+  const [body, setBody] = useState(null);
+
+  const handleClickOpen = () => {
+    setOpen(true);
+  };
+
+  const clearState = () => {
+    setTitle(null);
+    setTitle(null);
+  };
+
+  const handleClose = (isSaved = true) => {
```

Comment:
You chose a strange name for the argument, I think it would have been better to choose something like isConfirmed, because at this point it has not been saved.